google/cloud/storage/retry.py

```diff
@@ -24,7 +24,7 @@
 try:
     _RETRYABLE_STDLIB_TYPES = (ConnectionError,)
 except NameError:
-    _RETRYABLE_STDLIB_TYPES = ()
+    _RETRYABLE_STDLIB_TYPES = ()  # type: ignore
 
 
 _RETRYABLE_TYPES = _RETRYABLE_STDLIB_TYPES + (
```

Comment:
We would ideally be declaring types for these tuples, e.g. (before the `try:` here):

```python
_RETRYABLE_STDLIB_TYPES: Tuple[Exception]
_RETRYABLE_TYPES: Tuple[Exception]
```